cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,138 @@
+
+{{site.data.alerts.callout_danger}}
+The following list is provided only to illustrate a small subset of the measures you might need to take to ensure compliance.
+{{site.data.alerts.end}}
+
+- Ensure that your cluster is a {{ site.data.products.dedicated }} cluster.
+- Single Sign-On (SSO) helps you avoid storing user passwords in {{ site.data.products.db }}:
+  - [Cloud Organization Single Sign-On (SSO)](configure-cloud-org-sso.html), which allows members of your CockroachDB Cloud organization to authenticate using an identity from an identity provider (IdP) instead of using an email address and password.
+  - [Cluster SSO](cloud-sso-sql.html), which allows users to access the SQL interface of a CockroachDB cluster (whether provisioned on CockroachDB Cloud or self-hosted) with the full security of Single Sign-On (SSO), and the convenience of being able to choose from a variety of SSO identity providers, including CockroachDB Cloud, Google, Azure, GitHub, or your own self-hosted OIDC.
+- Enable [Customer-Managed Encryption Keys (CMEK)](cmek.html), which allow you to protect data at rest in a CockroachDB Dedicated cluster using a cryptographic key that is entirely within your control, hosted in a supported key-management system (KMS) platform.
+- Protect your cluster's network from the public internet by enabling the following features:
+  - [Private Clusters](private-clusters.html), which ensure that cluster nodes have no public IP addresses and that egress traffic moves over private subnets and through a highly-available NAT gateway that is unique to the cluster.
+  - [Egress Perimeter Controls](egress-perimeter-controls.html), which ensure that cluster egress operations, such as cluster backups, are restricted to a list of specified external destinations.
+- Protect data that is written from your cluster to storage on your cloud provider by enabling and using the following features:
+  - [Cluster log export](export-logs.html), which automatically exports logs from all nodes in all regions of your CockroachDB Dedicated cluster to your chosen cloud log sink. You must enable redaction of sensitive data in the log export configuration.
+  - [Cloud Organization audit log export](cloud-org-audit-logs.html), which captures audit logs when many types of events occur in your {{ site.data.products.db }} organization, such as when a cluster is created or when a user is added to or removed from an organization.
+  - Your cloud provider's [IAM roles](../stable/cloud-storage-authentication.html), which provide an extra layer of authentication when your cluster or {{ site.data.products.db }} write data to your cloud provider, such as managed backups or [change data capture (CDC)](../stable/change-data-capture-overview.html) operations.
```

Comment:
This is not needed.

Reply:
This was from your Slack message in https://cockroachlabs.slack.com/archives/C03GUEQ308N/p1672369958310469?thread_ts=1672342416.894089&cid=C03GUEQ308N. Did I misunderstand?